### PR TITLE
Support using SENTRY_RELEASE environment variable

### PR DESCRIPTION
**Pull request by challgren**
According to the [PHP SDK docs](https://docs.sentry.io/platforms/php/configuration/options/#release) a user can set the `SENTRY_RELEASE` environment variable. But with the Laravel specific package the `release` option is commented out and not using the default value.

### CODE REVIEW

**Comment by challgren** [comment]
Related https://github.com/getsentry/sentry-laravel/pull/51 and https://github.com/getsentry/sentry-laravel/issues/167

**Comment by stayallive** [comment]
What exactly is not working with setting `SENTRY_RELEASE` in your `.env`? It works just fine without any changes.

Also this PR enables running a shell command by default which we are not doing for a few reasons (mainly security and performance).

File: config/sentry.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     'dsn' => env('SENTRY_LARAVEL_DSN', env('SENTRY_DSN')),
 
     // capture release as git sha
-    // 'release' => trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD')),
+    'release' => env('SENTRY_RELEASE', trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD'))),
```
Comment (stayallive) [inline config/sentry.php:8]: I would be fine with this, but I am not with executing a shell command on our users machines by default, even though it looks harmless enough. Not everyone is using git or config caching which can cause issues.

```suggestion
    'release' => env('SENTRY_RELEASE'),
```

**Review by stayallive — CHANGES_REQUESTED** [review]
@cleptric I'm not sure we even want to keep the `git` command example in the config file, it's been around since forever and it's been kept but this might be better documented elsewhere because this is not the most efficient way to achieve this anyway.

File: config/sentry.php (hunk 1 above)
Comment (cleptric) [reply config/sentry.php:8]: I would also rather have this removed and go ahead with your suggestion here.

**Review by cleptric — APPROVED** [review]

**Pull request merged**